poll/public/handlebars/poll_studio.handlebars

```diff
@@ -2,21 +2,26 @@
 {{#each items}}
 <li class="field comp-setting-entry is-set poll-{{noun}}-studio-item">
     <div class="wrapper-comp-setting">
+        <div class="poll-move">
+            <button class="poll-move-up">&#9650;<span class="sr">&nbsp;move poll up</span></button>
+            <button class="poll-move-down">&#9660;<span class="sr">&nbsp;move poll down</span></button>
+        </div>
+        <button class="button action-button poll-delete-answer">Delete</button>
         <label class="label setting-label poll-setting-label" for="{{noun}}-label-{{key}}">{{noun}}</label>
         <input class="input setting-input" name="{{noun}}-label-{{key}}" id="{{noun}}-label-{{key}}" value="{{text}}" type="text" /><br />
         {{#if image}}
         <label class="label setting-label" for="{{noun}}-img-{{key}}">Image URL</label>
-        <input class="input setting-input" name="{{noun}}-img-{{key}}" id="{{noun}}-img-{{key}}" value="{{img}}" type="text" />
+        <input class="input setting-input" name="{{noun}}-img-{{key}}" id="{{noun}}-img-{{key}}" value="{{img}}" type="text" /><br />
+        <label class="label setting-label" for="{{noun}}-img_alt-{{key}}">Image alternate text</label>
+        <input class="input setting-input" name="{{noun}}-img_alt-{{key}}" id="{{noun}}-img_alt-{{key}}" value="{{img_alt}}" type="text" /><br />
```

Comment:
@sarina By "becoming invalid" I meant invalid when edited again – it wasn't worded very precisely.  People trying to edit an unrelated setting for a poll will suddenly get a validation error for something they didn't touch.
The other concern is that this XBlock is not only used on edx.org, but also in contexts with less strict accessibility requirements where enforcing the alt tags makes less sense.  Since this is a hard requirement for edx.org, we can go with the solution @antoviaque suggested.